src/lib/config/index.ts

```diff
@@ -67,6 +69,11 @@ export class Config {
     }
   }
 
+  objectExists(key: string): boolean {
+    const v = this.config[key];
+    return v !== null && typeof v === "object";
```

Comment:
I think this needs to be undefined, as `[]` will return undefined if there isn't a value present.
```suggestion
    return v !== undefined && typeof v === "object";
```